docs/source-pytorch/accelerators/gpu_intermediate.rst

```diff
@@ -95,54 +93,12 @@ project module) you can use the following method:
 
 We STRONGLY discourage this use because it has limitations (due to Python and PyTorch):
 
-1. The model you pass in will not update. Please save a checkpoint and restore from there.
-2. Set Dataloader(num_workers=0) or it will bottleneck training.
+1. After ``.fit()``, only the model's weights get restored to the main process, but no other state of the Trainer.
+2. Does not support multi-node training.
+3. It is generally slower than DDP.
 
-`ddp` is MUCH faster than `ddp_spawn`. We recommend you
-
-1. Install a top-level module for your project using setup.py
```

Comment:
This is no longer a requirement for a very very long time. Asking the user to do these extra steps is not necessary. Bottom line, if you can run your script as python x/y/z/train.py, it will work with DDP as well!